src/components/SignUp/Inputs/InputEmail.js

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+
+const InputEmail = (props) => {
```

Comment:
Question here, is there no way to reuse the input components from sign-in? If not, what are the differences?

Reply:
We can refactor the code for more reuse in another story